internal/repositories/github/service.go

```diff
@@ -440,3 +447,52 @@ func (r *repositoryService) persistRepository(
 
 	return outid, pbr, nil
 }
+
+func pbRepoFromProperties(
+	repoProperties *properties.Properties,
+	githubHook *github.Hook,
+	hookUUID string,
+) (*pb.Repository, error) {
+	name, err := repoProperties.GetProperty("name").AsString()
```

Comment:
Ozz suggested that we should have named constants for the common properties and after having written this code I agree